charts/emissary-ingress/templates/ambassador-agent.yaml

```diff
@@ -240,8 +240,28 @@ spec:
           value: {{ .Values.agent.rpcAddress }}
         - name: AES_SNAPSHOT_URL
           value: "http://{{ include "ambassador.fullname" . }}-admin.{{ include "ambassador.namespace" . }}:{{ .Values.adminService.snapshotPort }}/snapshot-external"
-  {{ if .Values.progressDeadlines }}
-  {{ if hasKey .Values.progressDeadlines "agent" }}
+      {{- with .Values.agent.resources }}
+        resources:
+          {{- toYaml . | nindent 10 }}
+      {{- end -}}
```

Comment:
Prefer to avoid `-}}` and only use `{{-`.  There's no need for the `-}}` here since the next thing has `{{-`